### ISSUE TITLE
grunt_scaffold missing in master

### ISSUE THREAD

**Issue by premasagar** [premasagar/sqwidget#107]
In master branch:

```
$ npm install
...
sh: ./grunt-scaffold/install.sh: No such file or directory
```

From package.json

``` json
"scripts": {
    "prepublish": "make dist",
    "postinstall": "./grunt-scaffold/install.sh",
    "test": "grunt test:continuous"
  }
```

**Comment by adhipg** [comment]
We don't need `postinstall` now since we're not using the scaffolding anymore.

**Comment by premasagar** [comment]
OK, I'll remove from package.json

**Comment by premasagar** [comment]
Fixed in 7edc6d2199
